modules/xmpp/JingleSessionPC.js

```diff
@@ -601,6 +601,14 @@ export default class JingleSessionPC extends JingleSession {
                         MediaSessionEvents.REMOTE_VIDEO_CONSTRAINTS_CHANGED, this);
                 });
         }
+
+        // Reconfigure the max bitrate every time the send resolution is changed
+        // on the sender in p2p case.
+        if (this.isP2P) {
+            this.rtc.addEventListener(
+                RTCEvents.LOCAL_TRACK_MAX_ENABLED_RESOLUTION_CHANGED,
```

Comment:
Isn't it the `TPC` class that emits the `RTCEvents.LOCAL_TRACK_MAX_ENABLED_RESOLUTION_CHANGED`? maybe it's better to have this check/sync up there. TPC also knows if it's p2p or not.

Reply:
Yep changed it to call it within TPC.